sphinx/sphinx.gradle

```diff
@@ -110,4 +113,7 @@ tasks.named('cppClientDocsPrepareDocker').configure {
 
 makePyDocTask('cppExamplesDocs', 'cpp-examples', [
         ("$rootDir/cpp-examples/doc".toString()):'source',
-], "$buildDir/cppExampleDocs")
+        ("$buildDir/cppClientDocs".toString()):'cpp'
+], "$buildDir/cppExampleDocs") {
```

Comment:
For consistency with all the other directories that end up in `build` directory the we might want to call this "$buildDir/cppExamplesDocs"

with an "s" in Examples

Reply:
In screenshot form, we can see why this one directory stands out
![image](https://user-images.githubusercontent.com/720660/135949551-b0c689e5-b061-460b-aa5d-eee3b2596735.png)